src/ssort/_ast.py

```diff
@@ -365,6 +365,60 @@ def _iter_child_nodes_of_index(node: ast.Index) -> Iterable[ast.AST]:
         yield node.value
 
 
+@iter_child_nodes.register(ast.Load)
+@iter_child_nodes.register(ast.Store)
+@iter_child_nodes.register(ast.Del)
+def _iter_child_nodes_of_expr_context(
+    node: ast.expr_context,
+) -> Iterable[ast.AST]:
+    return ()
```

Comment:
Perhaps we should explicitly skip these in the tests.  I think they are really attributes of the parent node.

Reply:
Yes, I think you're right. In fact, I think all the operators can be skipped as well.